myanimelist/media.py

```diff
@@ -119,136 +97,84 @@ def parse_sidebar(self, media_page):
     if error_tag:
         raise InvalidMediaError(self.id)
 
-    try:
-      title_tag = media_page.find(u'div', {'id': 'contentWrapper'}).find(u'h1')
-      if not title_tag.find(u'div'):
-        # otherwise, raise a MalformedMediaPageError.
-        raise MalformedMediaPageError(self.id, media_page, message="Could not find title div")
-    except:
-      if not self.session.suppress_parse_exceptions:
-        raise
-
-    try:
-      utilities.extract_tags(title_tag.find_all())
-      media_info[u'title'] = title_tag.text.strip()
-    except:
-      if not self.session.suppress_parse_exceptions:
-        raise
+    title_tag = media_page.find(u'div', {'id': 'contentWrapper'}).find(u'h1')
+    if not title_tag.find(u'div'):
+      # otherwise, raise a MalformedMediaPageError.
+      raise MalformedMediaPageError(self.id, media_page, message="Could not find title div")
+
+    utilities.extract_tags(title_tag.find_all())
+    media_info[u'title'] = title_tag.text.strip()
```

Comment:
Is there any particular reason you've removed all the try-except blocks here? The point of these blocks is to allow users to suppress parsing exceptions if they've set `suppress_parse_exceptions=True` in the current session. If you want to raise exceptions e.g. during debugging, you should just set this flag to False.